packages/react/src/components/DetailsList/DetailsRow.base.tsx

```diff
@@ -363,7 +363,7 @@ export class DetailsRowBase extends React.Component<IDetailsRowBaseProps, IDetai
         <GroupSpacer
           indentWidth={indentWidth}
           role="gridcell"
-          count={groupNestingDepth! - (this.props.collapseAllVisibility === CollapseAllVisibility.hidden ? 1 : 0)}
+          count={groupNestingDepth! === 0 ? -1 : groupNestingDepth!}
```

Comment:
This is the simplified result of swapping out `this.props.collapseAllVisibility === CollapseAllVisibility.hidden` for `groupNestingDepth! > 0`, similar to the change in DetailsHeader.